docs/how_to_guides/validation/how_to_deploy_a_scheduled_checkpoint_with_cron.rst

```diff
@@ -0,0 +1,60 @@
+###################################################
+How to deploy a scheduled Checkpoint with cron
+####################################################
+
+In this guide you'll learn how to deploy a `Checkpoint <link here>_` on a schedule using cron.
```

Comment:
Minor nit, how-tos aren't about teaching or learning. "This guide will show you how to..." "This guide will walk you through the steps..." "To {do X}, follow the steps in this guide..."